presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplitSource.java

```diff
@@ -56,16 +58,20 @@ public class IcebergSplitSource
     private final double minimumAssignedSplitWeight;
     private final ConnectorSession session;
 
+    private final TupleDomain<IcebergColumnHandle> infoColumnConstraints;
```

Comment:
Maybe we should change the name to `metadataColumnConstraints` to keep consistency as @yingsu00 mentioned.